test/Infrastructure.EFIntegration.Test/Helpers/DatabaseOptionsFactory.cs

```diff
@@ -10,16 +15,39 @@ public static class DatabaseOptionsFactory
 
     static DatabaseOptionsFactory()
     {
+        var services = new ServiceCollection()
```

Comment:
❓ Is all this needed given the above fixture setup?

Reply:
Yeah it is, this is for the EF Repositories where they use an inner `IServiceProvider` to get the `IDataProtectionProvider` but the fixture change is so that it's injected in the constructor for the Dapper repos.